imapclient/client_test.go

```diff
@@ -14,13 +15,27 @@ import (
 const (
 	testUsername = "test-user"
 	testPassword = "test-password"
+
+	simpleEml = `MIME-Version: 1.0
+	Message-Id: <191101702316132@example.com>
```

Comment:
This leaves a tab before the message header fields, which parsers will choke on.